tslint.json

```diff
@@ -1,6 +1,17 @@
 {
-	"rules": {
-		"indent": [true, "tabs"],
-		"semicolon": [true, "always"]
-	}
+    "extends": "tslint:recommended",
+    "rules": {
+		"encoding": true,
+		"indent": [true, "tabs", 2],
+		"linebreak-style": [true, "LF"],
+		"semicolon": [true, "always"],
+		"newline-before-return": false,
+		"no-consecutive-blank-lines": [true, 2],
+		"no-trailing-whitespace": true,
+		"no-var-requires": false,
+		"quotemark": [true, "single"],
+		"arrow-parens": true,
+		"eofline": true,
+		"no-console": false
+    }
```

Comment:
unrelated to "adding test" PR

Reply:
it is not directly related but it helps improving code quality.

Reply:
it means that we will need to discuss the parameters in this PR too.

Reply:
we should discuss that at some point but I don't see how that holds this PR back. We agreed that we need to define a set of rules to use in all our projects. Whenever we did that we can raise issues and update it.